packages/java/src/browser/java-client-contribution.ts

```diff
@@ -52,7 +53,8 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
         @inject(Window) protected readonly window: Window,
         @inject(CommandService) protected readonly commandService: CommandService,
         @inject(StatusBar) protected readonly statusBar: StatusBar,
-        @inject(SemanticHighlightingService) protected readonly semanticHighlightingService: SemanticHighlightingService
+        @inject(SemanticHighlightingService) protected readonly semanticHighlightingService: SemanticHighlightingService,
+        @inject(PreferenceService) protected readonly preferenceService: PreferenceService
```

Comment:
If you are adding new injections, it's best to use [property injection over constructor injection](https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection).
Adding new constructor injections is a breaking change.

Reply:
@vince-fugnitto 
Sorry. I fixed to use property injection.
https://github.com/theia-ide/theia/pull/5074/commits/bf5f0d929cde375804e6f442945ccfbe16a343e9